unit_tests/TestUtils.h

```diff
@@ -149,7 +149,7 @@ struct TestParams {
     static bool skipSerialTests;
 
     static void checkArgs([[maybe_unused]] int argc, [[maybe_unused]] char* argv[]) {
-        skipSerialTests = true;
+        skipSerialTests = false;
```

Comment:
Did you change this for testing and then forgot to change it back or do we want this in the dev branch?

Reply:
yes I changed it for testing purposes. There was also a short discussion about this flag in the element chat. Maybe we should discuss it in the next ippl meeting.

Reply:
Yes we decided not to skip it. So the false flag is fine.